app/main.py

```diff
@@ -1,2 +1,63 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: any
```

Comment:
The `value` attribute should use `Any` from the `typing` module instead of `any`. Change `value: any` to `value: Any`.